pkg/loghttp/push/otlp.go

```diff
@@ -147,6 +148,7 @@ func otlpToLokiPushRequest(ld plog.Logs, userID string, tenantsRetention Tenants
 
 		resourceAttributesAsStructuredMetadataSize := labelsSize(resourceAttributesAsStructuredMetadata)
 		stats.StructuredMetadataBytes[tenantsRetention.RetentionPeriodFor(userID, lbs)] += int64(resourceAttributesAsStructuredMetadataSize)
+		stats.ResourceAndSourceMetadataLabels[tenantsRetention.RetentionPeriodFor(userID, lbs)] = append(stats.ResourceAndSourceMetadataLabels[tenantsRetention.RetentionPeriodFor(userID, lbs)], resourceAttributesAsStructuredMetadata...)
```

Comment:
`tenantsRetention.RetentionPeriodFor()` seems somewhat complex, so this should be both easier to read and somewhat better performing?

```suggestion
		retentionPeriodForUser := tenantsRetention.RetentionPeriodFor(userID, lbs)
		stats.StructuredMetadataBytes[retentionPeriodForUser] += int64(resourceAttributesAsStructuredMetadataSize)
		stats.ResourceAndSourceMetadataLabels[retentionPeriodForUser] = append(stats.ResourceAndSourceMetadataLabels[retentionPeriodForUser], resourceAttributesAsStructuredMetadata...)
```